lib/index.js

```diff
@@ -31,7 +31,7 @@ var settings = [
   'partials',
   'pattern',
   'rename',
-  'exposeConsolidate'
+  'renderer',
```

Comment:
The comma isn't a mistake. I personally like this because it is more consistent. I don't think that we have to support older versions of node.